packages/design-system/src/components/select.tsx

```diff
@@ -135,20 +136,18 @@ const SelectItemBase = (
 type SelectItemProps = SelectPrimitive.SelectItemProps & {
   children: ReactNode;
 };
-export const SelectItem = React.forwardRef(SelectItemBase);
+export const SelectItem = forwardRef(SelectItemBase);
 
 export type SelectOption = string;
 
 export type SelectProps<Option = SelectOption> = Omit<
-  React.ComponentProps<typeof StyledTrigger>,
+  ComponentProps<typeof StyledTrigger>,
   "onChange" | "value" | "defaultValue"
 > & {
   options: Option[];
   defaultValue?: Option;
   value?: Option;
   onChange?: (option: Option) => void;
-  open?: boolean;
-  onOpenChange?: (open: boolean) => void;
```

Comment:
now we can't control the component from outside if we need to

Reply:
We may never need to. Unused features often just add complexity